source/manifests/aws/kustomization.yaml.tpl

```diff
@@ -2,13 +2,13 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-{{- $version := semver .metadata.distributionVersion }}
+{{- $version := semver .spec.distributionVersion }}
 
 resources:
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/cluster-autoscaler/v" $version.Major "." $version.Minor ".x" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/ebs-csi-driver" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/load-balancer-controller" }}
-  - {{ print "../" .common.relativeVendorPath "/katalog/aws/node-termination-handler" }}
+  - {{ print "../" .spec.distribution.common.relativeVendorPath "/modules/aws/katalog/cluster-autoscaler/v" $version.Major "." $version.Minor ".x" }}
```

Comment:
Why this ended up under a `modules/` directory?

Reply:
I don't know if I can link there the notion page but on the specs for the download phase you can see the folders tree is:
./vendor/modules
./vendor/installers
./vendor/bin